chpl/chpl-api/e2e/collections/search-view-controller.postman_collection.json

```diff
@@ -0,0 +1,143 @@
+{
+	"info": {
+		"_postman_id": "cbf3ffef-47da-4944-b0cf-2e81ead99247",
+		"name": "search-view-controller",
+		"schema": "https://schema.getpostman.com/json/collection/v2.1.0/collection.json"
+	},
+	"item": [
+		{
+			"name": "GET /data/svap- valid API Key",
+			"event": [
+				{
+					"listen": "test",
+					"script": {
+						"id": "be6a49d6-0ead-407b-acb0-44f80dc2d61c",
+						"exec": [
+							"pm.test(\"Valid api key header should return 200 status for SVAP\", function () {\r",
```

Comment:
Can we add one more test to test response body schema?
Since this is new end point better to have more test than only 200.

Reply:
added new test for validating svap schema